config/database.yml

```diff
@@ -1,12 +1,13 @@
 default: &default
   adapter:  "mysql2"
 
-  host:     <%= ENV['VELUM_DB_HOST']     || "localhost" %>
-  port:     <%= ENV['VELUM_DB_PORT']     || 0           %>
-  username: <%= ENV['VELUM_DB_USERNAME'] || "root"      %>
-  password: <%= ENV['VELUM_DB_PASSWORD'] || ""          %>
+  host:     <%= ENV['VELUM_DB_HOST']     || ""                            %>
+  port:     <%= ENV['VELUM_DB_PORT']     || 0                             %>
+  username: <%= ENV['VELUM_DB_USERNAME'] || "root"                        %>
+  password: <%= ENV['VELUM_DB_PASSWORD'] || ""                            %>
   encoding: utf8
-  pool:     <%= ENV['VELUM_DB_POOL']     || 5           %>
+  pool:     <%= ENV['VELUM_DB_POOL']     || 5                             %>
+  socket:   <%= ENV['VELUM_DB_SOCKET']   || "/var/run/mysqld/mysqld.sock" %>
```

Comment:
`VELUM_DB_SOCKET` will still point to the socket even when `VELUM_DB_HOST` is set.  Isn't this going to cause issues to Rails?

Wouldn't it be better to have something like:

```ruby
<% if ENV['VELUM_DB_SOCKET'] %>
socket:   <%= ENV['VELUM_DB_SOCKET'] %>
<% endif %>
```